.github/workflows/stale.yml

```
@@ -25,5 +25,5 @@ jobs:
           days-before-issue-stale: -1
           stale-issue-label: "Cleanup Flagged"
           remove-issue-stale-when-updated: false
-          exempt-pr-labels: "RED LABEL,Good First PR"
+          exempt-pr-labels: "RED LABEL,Good First PR,Stale-b-gone"
```

```suggestion
          exempt-pr-labels: "Good First PR,Upstream PR Merged,Stale-b-gone,Test Merged"
```
Good catch
Looks like some others got lost in the process of a prior TG update, if you could add those too